docs/articles/toc.yml

```diff
@@ -202,3 +202,5 @@
     href: utilities/circuit-breaker.md
 - name: Configurations
   href: configuration/toc.yml
+- name: Serialization
+  href: serialization/toc.yml
```

Comment:
We should probably migrate some stuff to this section

Reply:
There's a plan on moving Hyperion to its own page, so I think this section would host it.